package.json

```diff
@@ -11,12 +11,15 @@
     "astro": "astro"
   },
   "dependencies": {
-    "astro": "^4.15.3"
+    "astro": "^5.3.1"
   },
   "devDependencies": {
-    "@astrojs/tailwind": "^5.1.0",
+    "@astrojs/tailwind": "^6.0.0",
+    "@fontsource-variable/inter": "^5.1.1",
     "@fontsource-variable/space-grotesk": "^5.1.1",
     "@iconify-json/lucide": "^1.2.25",
+    "@iconify-json/simple-icons": "^1.2.26",
+    "@microflash/rehype-figure": "^2.1.3",
```

Comment:
- Bump to Astro v5, but stick to Tailwind v3 for now
- Add `Inter` as new `sans` font
- Add `@iconify-json/simple-icons` to handle `SocialShare`'s icons
- Add `@microflash/rehype-figure` to have (a crutch for) image captions and an (easier) way to pull article images into the layout gutters on larger screens, and give them some additional space to benefit from regarding readability: <img width="1332" alt="image" src="https://github.com/user-attachments/assets/9bd92d88-10c9-4e73-bdb8-3016e1bc37fa" />